[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building easylp 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ easylp ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ easylp ---
[INFO] Changes detected - recompiling the module!
[INFO] Using Groovy-Eclipse compiler to compile both Java and Groovy files
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 11.420 s
[INFO] Finished at: 2019-01-23T11:29:00-05:00
[INFO] Final Memory: 23M/1131M
[INFO] ------------------------------------------------------------------------
Classpath found cached, skipping classpath build.
1    [main] INFO  org.linqs.psl.config.ConfigManager  - Configuration file loaded: file:/network/rit/home/aa465948/git/adv_csl/psl/psl-sybils1/easy/groovy/target/classes/psl.properties
52   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.name. Returning default of default.
57   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.dbpath. Returning default of /tmp.
57   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.data.path. Returning default of ../data.
69   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.output.outputdir. Returning default of output/default.
77   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.adjacent. Returning default of 1.
78   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.benign. Returning default of 1.
78   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.sybils. Returning default of 1.
651  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.valuecolumn. Returning default of truth.
651  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.confidencecolumn. Returning default of confidence.
651  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.partitioncolumn. Returning default of partition.
776  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.usestringids. Returning default of true.
780  [main] DEBUG org.linqs.psl.database.rdbms.RDBMSDataStore  - Registered 0 pre-existing predicates from RDBMS.
866  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Running experiment default
869  [main] WARN  org.linqs.psl.database.rdbms.RDBMSDataStoreMetadata  - Determining max partition, no partitions found null
909  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Defining model rules
973  [main] DEBUG org.linqs.psl.example.sybils1.Sybils1  - model: Model:
1.0: ( ADJACENT(E1, E2) & BENIGN(E1) ) >> BENIGN(E2) ^2
1.0: ( ADJACENT(E1, E2) & ~( BENIGN(E1) ) ) >> ~( BENIGN(E2) ) ^2
1.0: ( ADJACENT(E1, E2) & SYBILS(E1) ) >> SYBILS(E2) ^2
1.0: ( ADJACENT(E1, E2) & ~( SYBILS(E1) ) ) >> ~( SYBILS(E2) ) ^2
1.0: BENIGN(E1) >> ~( SYBILS(E1) ) ^2
973  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Loading data into database
org.h2.message.DbException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
org.h2.message.DbException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.TraceSystem.logWritingError(TraceSystem.java:286)
	at org.h2.message.TraceSystem.openWriter(TraceSystem.java:307)
	at org.h2.message.TraceSystem.writeFile(TraceSystem.java:255)
	at org.h2.message.TraceSystem.write(TraceSystem.java:239)
	at org.h2.message.Trace.error(Trace.java:182)
	at org.h2.engine.Database.setBackgroundException(Database.java:2037)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.write(MVStore.java:843)
	at org.h2.mvstore.MVStore.storeNowTry(MVStore.java:1204)
	at org.h2.mvstore.MVStore.storeNow(MVStore.java:1048)
	at org.h2.mvstore.MVStore.commitAndSave(MVStore.java:1037)
	at org.h2.mvstore.MVStore.commit(MVStore.java:998)
	at org.h2.mvstore.db.TransactionStore.endTransaction(TransactionStore.java:467)
	at org.h2.mvstore.db.TransactionStore.commit(TransactionStore.java:364)
	at org.h2.mvstore.db.TransactionStore$Transaction.commit(TransactionStore.java:779)
	at org.h2.engine.Session.commit(Session.java:611)
	at org.h2.command.Command.stop(Command.java:156)
	at org.h2.command.Command.executeUpdate(Command.java:288)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:160)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:146)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insertInternal(RDBMSDataLoader.java:202)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insert(RDBMSDataLoader.java:142)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSInserter.insert(RDBMSDataLoader.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:87)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.file.LoadDelimitedData.loadTabData(LoadDelimitedData.java:52)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:100)
	at org.linqs.psl.utils.dataloading.InserterUtils$loadDelimitedData.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
	at org.linqs.psl.example.sybils1.Sybils1.loadData(Sybils.groovy:152)
	at org.linqs.psl.example.sybils1.Sybils1.this$2$loadData(Sybils.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:151)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:179)
	at org.linqs.psl.example.sybils1.Sybils1.run(Sybils.groovy:238)
	at org.linqs.psl.example.sybils1.Sybils1$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:114)
	at org.linqs.psl.example.sybils1.Sybils1.main(Sybils.groovy:268)
Caused by: org.h2.jdbc.JdbcSQLException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 51 more
Caused by: java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:101)
	at org.h2.store.fs.FilePathDisk.newOutputStream(FilePathDisk.java:290)
	at org.h2.store.fs.FileUtils.newOutputStream(FileUtils.java:233)
	at org.h2.message.TraceSystem.openWriter(TraceSystem.java:303)
	... 48 more
81642 [main] ERROR org.linqs.psl.database.rdbms.RDBMSDataLoader  - General error: "java.lang.IllegalStateException: Writing to cache:nio:/tmp/Sybils1.mv.db failed; length 868352 at 1764319232 [1.4.192/2]" [50000-192]
[17, 408]
Exception in thread "main" java.lang.AssertionError: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: Writing to cache:nio:/tmp/Sybils1.mv.db failed; length 868352 at 1764319232 [1.4.192/2]" [50000-192]
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insertInternal(RDBMSDataLoader.java:206)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insert(RDBMSDataLoader.java:142)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSInserter.insert(RDBMSDataLoader.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:87)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.file.LoadDelimitedData.loadTabData(LoadDelimitedData.java:52)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:100)
	at org.linqs.psl.utils.dataloading.InserterUtils$loadDelimitedData.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
	at org.linqs.psl.example.sybils1.Sybils1.loadData(Sybils.groovy:152)
	at org.linqs.psl.example.sybils1.Sybils1.this$2$loadData(Sybils.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:151)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:179)
	at org.linqs.psl.example.sybils1.Sybils1.run(Sybils.groovy:238)
	at org.linqs.psl.example.sybils1.Sybils1$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:114)
	at org.linqs.psl.example.sybils1.Sybils1.main(Sybils.groovy:268)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: Writing to cache:nio:/tmp/Sybils1.mv.db failed; length 868352 at 1764319232 [1.4.192/2]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.message.DbException.toSQLException(DbException.java:268)
	at org.h2.message.TraceObject.logAndConvert(TraceObject.java:352)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:151)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insertInternal(RDBMSDataLoader.java:202)
	... 28 more
Caused by: java.lang.IllegalStateException: Writing to cache:nio:/tmp/Sybils1.mv.db failed; length 868352 at 1764319232 [1.4.192/2]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:466)
	at org.h2.mvstore.FileStore.writeFully(FileStore.java:113)
	at org.h2.mvstore.MVStore.write(MVStore.java:841)
	at org.h2.mvstore.MVStore.storeNowTry(MVStore.java:1204)
	at org.h2.mvstore.MVStore.storeNow(MVStore.java:1048)
	at org.h2.mvstore.MVStore.commitAndSave(MVStore.java:1037)
	at org.h2.mvstore.MVStore.commit(MVStore.java:998)
	at org.h2.mvstore.db.TransactionStore.endTransaction(TransactionStore.java:467)
	at org.h2.mvstore.db.TransactionStore.commit(TransactionStore.java:364)
	at org.h2.mvstore.db.TransactionStore$Transaction.commit(TransactionStore.java:779)
	at org.h2.engine.Session.commit(Session.java:611)
	at org.h2.command.Command.stop(Command.java:156)
	at org.h2.command.Command.executeUpdate(Command.java:288)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:160)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:146)
	... 29 more
Caused by: java.io.IOException: No space left on device
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:66)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:89)
	at sun.nio.ch.IOUtil.write(IOUtil.java:65)
	at sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:778)
	at sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:764)
	at org.h2.store.fs.FileNio.write(FilePathNio.java:79)
	at org.h2.mvstore.cache.FilePathCache$FileCache.write(FilePathCache.java:129)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:462)
	... 43 more
[2019-01-23T11:30:23-0500]: Failed to run
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building easylp 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ easylp ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ easylp ---
[INFO] Changes detected - recompiling the module!
[INFO] Using Groovy-Eclipse compiler to compile both Java and Groovy files
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.636 s
[INFO] Finished at: 2019-01-23T11:30:29-05:00
[INFO] Final Memory: 23M/1435M
[INFO] ------------------------------------------------------------------------
Classpath found cached, skipping classpath build.
0    [main] INFO  org.linqs.psl.config.ConfigManager  - Configuration file loaded: file:/network/rit/home/aa465948/git/adv_csl/psl/psl-sybils1/easy/groovy/target/classes/psl.properties
53   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.name. Returning default of default.
58   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.dbpath. Returning default of /tmp.
58   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.data.path. Returning default of ../data.
68   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.output.outputdir. Returning default of output/default.
76   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.adjacent. Returning default of 1.
77   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.benign. Returning default of 1.
77   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.sybils. Returning default of 1.
503  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.valuecolumn. Returning default of truth.
503  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.confidencecolumn. Returning default of confidence.
503  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.partitioncolumn. Returning default of partition.
524  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.usestringids. Returning default of true.
528  [main] DEBUG org.linqs.psl.database.rdbms.RDBMSDataStore  - Registered 0 pre-existing predicates from RDBMS.
610  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Running experiment default
613  [main] WARN  org.linqs.psl.database.rdbms.RDBMSDataStoreMetadata  - Determining max partition, no partitions found null
653  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Defining model rules
702  [main] DEBUG org.linqs.psl.example.sybils1.Sybils1  - model: Model:
1.0: ( ADJACENT(E1, E2) & BENIGN(E1) ) >> BENIGN(E2) ^2
1.0: ( ADJACENT(E1, E2) & ~( BENIGN(E1) ) ) >> ~( BENIGN(E2) ) ^2
1.0: ( ADJACENT(E1, E2) & SYBILS(E1) ) >> SYBILS(E2) ^2
1.0: ( ADJACENT(E1, E2) & ~( SYBILS(E1) ) ) >> ~( SYBILS(E2) ) ^2
1.0: BENIGN(E1) >> ~( SYBILS(E1) ) ^2
702  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Loading data into database
org.h2.message.DbException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
org.h2.message.DbException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.TraceSystem.logWritingError(TraceSystem.java:286)
	at org.h2.message.TraceSystem.openWriter(TraceSystem.java:307)
	at org.h2.message.TraceSystem.writeFile(TraceSystem.java:255)
	at org.h2.message.TraceSystem.write(TraceSystem.java:239)
	at org.h2.message.Trace.error(Trace.java:182)
	at org.h2.engine.Database.setBackgroundException(Database.java:2037)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.write(MVStore.java:843)
	at org.h2.mvstore.MVStore.storeNowTry(MVStore.java:1204)
	at org.h2.mvstore.MVStore.storeNow(MVStore.java:1048)
	at org.h2.mvstore.MVStore.commitAndSave(MVStore.java:1037)
	at org.h2.mvstore.MVStore.commit(MVStore.java:998)
	at org.h2.mvstore.db.TransactionStore.endTransaction(TransactionStore.java:467)
	at org.h2.mvstore.db.TransactionStore.commit(TransactionStore.java:364)
	at org.h2.mvstore.db.TransactionStore$Transaction.commit(TransactionStore.java:779)
	at org.h2.engine.Session.commit(Session.java:611)
	at org.h2.command.Command.stop(Command.java:156)
	at org.h2.command.Command.executeUpdate(Command.java:288)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:160)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:146)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insertInternal(RDBMSDataLoader.java:202)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insert(RDBMSDataLoader.java:142)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSInserter.insert(RDBMSDataLoader.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:87)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.file.LoadDelimitedData.loadTabData(LoadDelimitedData.java:52)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:100)
	at org.linqs.psl.utils.dataloading.InserterUtils$loadDelimitedData.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
	at org.linqs.psl.example.sybils1.Sybils1.loadData(Sybils.groovy:152)
	at org.linqs.psl.example.sybils1.Sybils1.this$2$loadData(Sybils.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:151)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:179)
	at org.linqs.psl.example.sybils1.Sybils1.run(Sybils.groovy:238)
	at org.linqs.psl.example.sybils1.Sybils1$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:114)
	at org.linqs.psl.example.sybils1.Sybils1.main(Sybils.groovy:268)
Caused by: org.h2.jdbc.JdbcSQLException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 51 more
Caused by: java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:101)
	at org.h2.store.fs.FilePathDisk.newOutputStream(FilePathDisk.java:290)
	at org.h2.store.fs.FileUtils.newOutputStream(FileUtils.java:233)
	at org.h2.message.TraceSystem.openWriter(TraceSystem.java:303)
	... 48 more
1715 [main] ERROR org.linqs.psl.database.rdbms.RDBMSDataLoader  - General error: "java.lang.IllegalStateException: Writing to cache:nio:/tmp/Sybils1.mv.db failed; length 487424 at 1764319232 [1.4.192/2]" [50000-192]
[83819, 94461]
Exception in thread "main" java.lang.AssertionError: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: Writing to cache:nio:/tmp/Sybils1.mv.db failed; length 487424 at 1764319232 [1.4.192/2]" [50000-192]
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insertInternal(RDBMSDataLoader.java:206)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insert(RDBMSDataLoader.java:142)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSInserter.insert(RDBMSDataLoader.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:87)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.file.LoadDelimitedData.loadTabData(LoadDelimitedData.java:52)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:100)
	at org.linqs.psl.utils.dataloading.InserterUtils$loadDelimitedData.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
	at org.linqs.psl.example.sybils1.Sybils1.loadData(Sybils.groovy:152)
	at org.linqs.psl.example.sybils1.Sybils1.this$2$loadData(Sybils.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:151)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:179)
	at org.linqs.psl.example.sybils1.Sybils1.run(Sybils.groovy:238)
	at org.linqs.psl.example.sybils1.Sybils1$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:114)
	at org.linqs.psl.example.sybils1.Sybils1.main(Sybils.groovy:268)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: Writing to cache:nio:/tmp/Sybils1.mv.db failed; length 487424 at 1764319232 [1.4.192/2]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.message.DbException.toSQLException(DbException.java:268)
	at org.h2.message.TraceObject.logAndConvert(TraceObject.java:352)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:151)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insertInternal(RDBMSDataLoader.java:202)
	... 28 more
Caused by: java.lang.IllegalStateException: Writing to cache:nio:/tmp/Sybils1.mv.db failed; length 487424 at 1764319232 [1.4.192/2]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:466)
	at org.h2.mvstore.FileStore.writeFully(FileStore.java:113)
	at org.h2.mvstore.MVStore.write(MVStore.java:841)
	at org.h2.mvstore.MVStore.storeNowTry(MVStore.java:1204)
	at org.h2.mvstore.MVStore.storeNow(MVStore.java:1048)
	at org.h2.mvstore.MVStore.commitAndSave(MVStore.java:1037)
	at org.h2.mvstore.MVStore.commit(MVStore.java:998)
	at org.h2.mvstore.db.TransactionStore.endTransaction(TransactionStore.java:467)
	at org.h2.mvstore.db.TransactionStore.commit(TransactionStore.java:364)
	at org.h2.mvstore.db.TransactionStore$Transaction.commit(TransactionStore.java:779)
	at org.h2.engine.Session.commit(Session.java:611)
	at org.h2.command.Command.stop(Command.java:156)
	at org.h2.command.Command.executeUpdate(Command.java:288)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:160)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:146)
	... 29 more
Caused by: java.io.IOException: No space left on device
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:66)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:89)
	at sun.nio.ch.IOUtil.write(IOUtil.java:65)
	at sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:778)
	at sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:764)
	at org.h2.store.fs.FileNio.write(FilePathNio.java:79)
	at org.h2.mvstore.cache.FilePathCache$FileCache.write(FilePathCache.java:129)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:462)
	... 43 more
[2019-01-23T11:30:31-0500]: Failed to run
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building easylp 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ easylp ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ easylp ---
[INFO] Changes detected - recompiling the module!
[INFO] Using Groovy-Eclipse compiler to compile both Java and Groovy files
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.789 s
[INFO] Finished at: 2019-01-23T11:30:36-05:00
[INFO] Final Memory: 23M/1488M
[INFO] ------------------------------------------------------------------------
Classpath found cached, skipping classpath build.
0    [main] INFO  org.linqs.psl.config.ConfigManager  - Configuration file loaded: file:/network/rit/home/aa465948/git/adv_csl/psl/psl-sybils1/easy/groovy/target/classes/psl.properties
52   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.name. Returning default of default.
57   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.dbpath. Returning default of /tmp.
57   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.data.path. Returning default of ../data.
68   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.experiment.output.outputdir. Returning default of output/default.
77   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.adjacent. Returning default of 1.
78   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.benign. Returning default of 1.
78   [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.model.weights.sybils. Returning default of 1.
288  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.valuecolumn. Returning default of truth.
288  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.confidencecolumn. Returning default of confidence.
288  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.partitioncolumn. Returning default of partition.
309  [main] INFO  org.linqs.psl.config.ConfigManager  - No value found for option Sybils1.rdbmsdatastore.usestringids. Returning default of true.
313  [main] DEBUG org.linqs.psl.database.rdbms.RDBMSDataStore  - Registered 0 pre-existing predicates from RDBMS.
392  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Running experiment default
395  [main] WARN  org.linqs.psl.database.rdbms.RDBMSDataStoreMetadata  - Determining max partition, no partitions found null
435  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Defining model rules
485  [main] DEBUG org.linqs.psl.example.sybils1.Sybils1  - model: Model:
1.0: ( ADJACENT(E1, E2) & BENIGN(E1) ) >> BENIGN(E2) ^2
1.0: ( ADJACENT(E1, E2) & ~( BENIGN(E1) ) ) >> ~( BENIGN(E2) ) ^2
1.0: ( ADJACENT(E1, E2) & SYBILS(E1) ) >> SYBILS(E2) ^2
1.0: ( ADJACENT(E1, E2) & ~( SYBILS(E1) ) ) >> ~( SYBILS(E2) ) ^2
1.0: BENIGN(E1) >> ~( SYBILS(E1) ) ^2
485  [main] INFO  org.linqs.psl.example.sybils1.Sybils1  - Loading data into database
org.h2.message.DbException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
org.h2.message.DbException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.TraceSystem.logWritingError(TraceSystem.java:286)
	at org.h2.message.TraceSystem.openWriter(TraceSystem.java:307)
	at org.h2.message.TraceSystem.writeFile(TraceSystem.java:255)
	at org.h2.message.TraceSystem.write(TraceSystem.java:239)
	at org.h2.message.Trace.error(Trace.java:182)
	at org.h2.engine.Database.setBackgroundException(Database.java:2037)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.write(MVStore.java:843)
	at org.h2.mvstore.MVStore.storeNowTry(MVStore.java:1204)
	at org.h2.mvstore.MVStore.storeNow(MVStore.java:1048)
	at org.h2.mvstore.MVStore.commitAndSave(MVStore.java:1037)
	at org.h2.mvstore.MVStore.writeInBackground(MVStore.java:2481)
	at org.h2.mvstore.MVStore$BackgroundWriterThread.run(MVStore.java:2694)
Caused by: org.h2.jdbc.JdbcSQLException: Log file error: "/tmp/Sybils1.trace.db", cause: "java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)" [90034-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 15 more
Caused by: java.io.FileNotFoundException: /tmp/Sybils1.trace.db (No space left on device)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:101)
	at org.h2.store.fs.FilePathDisk.newOutputStream(FilePathDisk.java:290)
	at org.h2.store.fs.FileUtils.newOutputStream(FileUtils.java:233)
	at org.h2.message.TraceSystem.openWriter(TraceSystem.java:303)
	... 12 more
1763 [main] ERROR org.linqs.psl.database.rdbms.RDBMSDataLoader  - The database has been closed [90098-192]
[6612, 7311]
Exception in thread "main" java.lang.AssertionError: org.h2.jdbc.JdbcSQLException: The database has been closed [90098-192]
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insertInternal(RDBMSDataLoader.java:206)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insert(RDBMSDataLoader.java:142)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSInserter.insert(RDBMSDataLoader.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:87)
	at org.linqs.psl.utils.dataloading.InserterUtils$2.create(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.file.LoadDelimitedData.loadTabData(LoadDelimitedData.java:52)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:82)
	at org.linqs.psl.utils.dataloading.InserterUtils.loadDelimitedData(InserterUtils.java:100)
	at org.linqs.psl.utils.dataloading.InserterUtils$loadDelimitedData.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
	at org.linqs.psl.example.sybils1.Sybils1.loadData(Sybils.groovy:152)
	at org.linqs.psl.example.sybils1.Sybils1.this$2$loadData(Sybils.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:151)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:179)
	at org.linqs.psl.example.sybils1.Sybils1.run(Sybils.groovy:238)
	at org.linqs.psl.example.sybils1.Sybils1$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:110)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:114)
	at org.linqs.psl.example.sybils1.Sybils1.main(Sybils.groovy:268)
Caused by: org.h2.jdbc.JdbcSQLException: The database has been closed [90098-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Session.getTransaction(Session.java:1595)
	at org.h2.engine.Session.getStatementSavepoint(Session.java:1605)
	at org.h2.engine.Session.setSavepoint(Session.java:799)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:160)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:146)
	at org.linqs.psl.database.rdbms.RDBMSDataLoader$RDBMSTableInserter.insertInternal(RDBMSDataLoader.java:202)
	... 28 more
[2019-01-23T11:30:39-0500]: Failed to run
